docs/sources/whatsnew/whats-new-in-v9-3.md

```diff
@@ -0,0 +1,183 @@
+---
+_build:
+  list: false
+aliases:
+  - /docs/grafana/latest/guides/whats-new-in-v9-3/
+description: Feature and improvement highlights for Grafana v9.3
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '9.3'
+  - release notes
+title: What's new in Grafana v9.3
+weight: -33
+---
+
+# What’s new in Grafana v9.3 (Beta)
+
+Welcome to Grafana v9.3. If you’d prefer to dig into the details, check out the complete [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+## What's New feature template
+
+Use this template to add your what's new section.
+
+[Generally available | Available in experimental/beta] in Grafana [Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced]
+
+Description. Include an overview of the feature and problem it solves, and where to learn more (like a link to the docs).
+
+Add a screenshot to this enablement material folder and link to it here.
+
+## Geomap panel
+
+Generally available in Grafana Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced
+
+We have added a new alpha layer type in Geomap called photo layer. This layer enables you to render a photo at each data point. To learn more about the photo layer and the geomap panel, see the [documentation]({{< relref "../panels-visualizations/visualizations/geomap/#photos-layer-alpha" >}}).
+
+{{< figure src="/static/img/docs/geomap-panel/geomap-photos-9-3-0.png" max-width="750px" caption="Geomap panel photos layer" >}}
+
+## Canvas panel
+
+Available in beta in Grafana Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced
+
+We have added icon value mapping support. This enables you to dynamically set which icon to display based on your data. To learn more about the Canvas panel, see the [documentation]({{< relref "../panels-visualizations/visualizations/canvas" >}}).
+
+{{< video-embed src="/static/img/docs/canvas-panel/canvas-icon-value-mapping-support-9-3-0.mp4" max-width="750px" caption="Canvas panel icon value mapping support" >}}
```

Comment:
Hi @chri2547, I've added info for 9.3 release from edge squad. Thanks!

[Geomap image added here](https://github.com/grafana/website/pull/10847)

[Canvas video added here](https://github.com/grafana/website/pull/10852)